15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/change-rich-text-editor-ui.md

```diff
@@ -0,0 +1,57 @@
+# Change Rich Text Editor UI
+
+{% hint style="info" %}
+Umbraco 15 ships with two UI options for working with rich text, Tiptap and TinyMCE. The latter will eventually be removed from the CMS.
+
+This article will guide you through switching from using the TinyMCE UI to using the new Tiptap UI.
+{% endhint %}
+
+The following steps will guide you through changing the property editor used for rich text on an existing Document Type.
+
+## Create a new Data Type
+
+The first step in this guide is to create a new Rich Text Editor Data Type using the Tiptap UI.
+
+As an alternative, you can create a new Data Type with the Tiptap UI when updating the Document Type.
```

Comment:
Umm, I'm not sure if Tiptap UI is the correct term cz we are not seeing any user interface as such other than selecting a data type/property editor.